opentelemetry-otlp/src/exporter/http/logs.rs

```diff
@@ -19,7 +19,13 @@ impl LogExporter for OtlpHttpClient {
                 _ => Err(LogError::Other("exporter is already shut down".into())),
             })?;
 
-        let (body, content_type) = { self.build_logs_export_body(batch, &self.resource)? };
+        //TODO :avoid cloning when logdata is borrowed?
+        let owned_batch = batch
+            .iter()
+            .map(|&log_data| log_data.clone()) // Converts Cow to owned LogData
```

Comment:
There is no 🐮 now 🤣

Reply:
unfortunately, had to bring `her` back in latest commit. Too much milking needed in  batchprocessor without that.